doc/en/user/source/styling/workshop/ysld/linestring.rst

```diff
@@ -0,0 +1,671 @@
+.. _ysld.line:
+
+Lines
+=====
+
+We will start our tour of YSLD styling by looking at the representation of lines.
+
+.. figure:: ../style/img/LineSymbology.svg
+   
+   LineString Geometry
+
+Review of line symbology:
+
+* Lines can be used to represent either abstract concepts with length but not width such as networks and boundaries, or long thin features with a didth that is too smallt o represent on the map. This means that **the visual width of line symbols do not normally change depending on scale.**
+
+* Lines are recorded as LineStrings or Curves depending on the geometry model used.
+
+* SLD uses a **LineSymbolizer** to record how the shape of a line is drawn. The primary characteristic documented is the **Stroke** used to draw each segment between vertices.
+
+* Labeling of line work is anchored to the mid-point of the line. GeoServer provides a vendor option to allow label rotation aligned with line segments.
+
+For our exercises we are going to be using simple YSLD documents, often consisting of a single rule, in order to focus on the properties used for line symbology.
+
+Each exercise makes use of the ``ne:roads`` layer.
+
+Reference:
+
+* :ref:`Line Symbology <extensions/css/properties.html#line-symbology>` (User Manual | CSS Property Listing)
+* :ref:`Lines <extensions/css/cookbook/line.html>` (User Manual | CSS Cookbook)
+* :ref:`LineString <styling/sld-reference/linesymbolizer.html>` (User Manual | SLD Reference )
+
+Line
+----
+
+A line symbolizer is represented by a :kbd:`line` key.  You can make a completely default symbolizer by giving it an empty map
+
+.. code-block:: yaml
+   
+        line: {}
```

Comment:
Remove `{}`, turn this into valid YSLD. This was confusing most workshop attendees.